go.mod

```diff
@@ -344,17 +344,17 @@ require (
 )
 
 replace (
-	github.com/CosmWasm/wasmd => github.com/sei-protocol/sei-wasmd v0.2.0
+	github.com/CosmWasm/wasmd => github.com/sei-protocol/sei-wasmd v0.2.1-0.20240729160304-78b3acbb7d98
 	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
-	github.com/cosmos/cosmos-sdk => github.com/sei-protocol/sei-cosmos v0.3.26
+	github.com/cosmos/cosmos-sdk => github.com/sei-protocol/sei-cosmos v0.3.28-0.20240722133704-db06037d7c06
 	github.com/cosmos/iavl => github.com/sei-protocol/sei-iavl v0.1.9
-	github.com/cosmos/ibc-go/v3 => github.com/sei-protocol/sei-ibc-go/v3 v3.3.1
+	github.com/cosmos/ibc-go/v3 => github.com/sei-protocol/sei-ibc-go/v3 v3.0.0-20240721122450-c087f564be1d
```

Comment:
We shouldn't change ibc from v3.3.1 to v3.0.0 I think? We should probably release a new tag based on v3.3.1, @Kbhat1 could you take a look as well?